docs/DeveloperGuide.md

```diff
@@ -273,14 +273,15 @@ interview availability dates.
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​  | I want to …​                       | So that I can…​                                                               |
-|----------|----------|------------------------------------|-------------------------------------------------------------------------------|
-| `* * *`  | user     | add a candidate to the system      | keep track of their application details for consideration.                    |
-| `* * *`  | user     | delete a candidate from the system | remove entries that I no longer need.                                         |
-| `* * *`  | user     | find a candidate in the system     | access details of the candidate without having to go through the entire list. |
-| `* * *`  | user     | list all candidates in the system  | monitor the application pool.                                                 |
-| `*`      | user     | sort candidates                    | get familiarised with the system.                                             |
-| `* * *`  | new user | view all available commands        | get familiarised with the system.                                             |
+| Priority | As a …​  | I want to …​                          | So that I can…​                                                               |
+|----------|----------|---------------------------------------|-------------------------------------------------------------------------------|
+| `* * *`  | user     | add a candidate to the system         | keep track of their application details for consideration.                    |
+| `* * *`  | user     | delete a candidate from the system    | remove entries that I no longer need.                                         |
+| `* * *`  | user     | find a candidate in the system        | access details of the candidate without having to go through the entire list. |
+| `* * *`  | user     | list all candidates in the system     | monitor the application pool.                                                 |
+| `*`      | user     | sort candidates                       | reorder candidates based on a specific attribute field.                       |
+| `* * *`  | user     | schedule TA candidates for interviews | keep track of the interview schedule.                                         |
```

Comment:
I'm not sure if you would like to add the `schedule` command added here to the 'Command Summary' section in the User Guide in this PR? :) Else it can be done in a later commit/PR as well!!

Reply:
Alright I will remove it first! Give me a moment to commit the changes